datumaro/components/annotation.py

```diff
@@ -787,3 +789,14 @@ class Caption(Annotation):
 
     _type = AnnotationType.caption
     caption: str = field(converter=str)
+
+
+@attrs(slots=True, order=False)
+class ImageResolution(Annotation):
+    """
+    Represents high and low resolution images.
+    """
+
+    _type = AnnotationType.image_resolution
+    hr_image = field()
```

Comment:
```suggestion
    hr_image: Image
```
Why the definitions are different?

Reply:
Fixed